.gitignore

```diff
@@ -27,7 +27,6 @@ yarn-debug.log*
 yarn-error.log*
 
 # local env files
-.env.local
```

Comment:
Please, do not remove this file from gitignor.

Reply:
Another pattern that can be done is `.env.example`, which can then be copied with npm postinstall to `.env.local`, so `.env.example` comes pre-loaded with defaults and sample strings